doc(cloud-azure-management-discover): refresh doc page CTOR-1357 (#4699)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/cloud-azure-management-discover.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Pack assets

### Templates

> **Azure Discover** monitoring connector doesn't have host template.

Reviewer: ```suggestion
The **Azure Discover** monitoring connector doesn't bring any host templates.
```

### Discovery rules

#### Host discovery

| Rule name                           | Description                               |
|:------------------------------------|:------------------------------------------|
| Microsoft Azure Management Discover | Discover Microsoft Azure resources        |
| Azure Tenant Discovery              | Discover all Azure resources for a tenant |

More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).

### Collected metrics & status

No metrics or status for this monitoring connector.

## Prerequisites

Please find all the prerequisites needed for Centreon to get information from Azure on the [dedicated page](../getting-started/how-to-guides/azure-credential-configuration.md).

> ## Installer le connecteur de supervision

Reviewer: FR in EN

> ## Installer le connecteur de supervision

Author: ```suggestion
## Installing the monitoring connector
```

1. If the platform uses an *online* license, you can skip the package installation
instruction below as it is not required to have the connector displayed within the
**Configuration > Connectors > Monitoring Connectors** menu.
If the platform uses an *offline* license, install the package on the **central server**
with the command corresponding to the operating system's package manager:

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```bash
dnf install centreon-pack-cloud-azure-management-discover
```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```bash
dnf install centreon-pack-cloud-azure-management-discover
```

</TabItem>
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">

```bash
apt install centreon-pack-cloud-azure-management-discover
```

</TabItem>
<TabItem value="CentOS 7" label="CentOS 7">

```bash
yum install centreon-pack-cloud-azure-management-discover
```

</TabItem>
</Tabs>

> 2. Whatever the license type (*online* or *offline*), install the **Ansible Tower** connector through

Reviewer: Ansible Tower

> 2. Whatever the license type (*online* or *offline*), install the **Ansible Tower** connector through

Author: ```suggestion
2. Whatever the license type (*online* or *offline*), install the **Azure Discover** connector through
```

You still have to manually install the plugin on the poller(s) when:
- Automatic plugin installation is turned off
- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet

> More information in the [Installing the plugin](/docs/monitoring/pluginpacks/#installing-the-plugin) section.

Use the commands below according to your operating system's package manager:

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```bash
dnf install centreon-plugin-Cloud-Azure-Management-Discover-Api
```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```bash
dnf install centreon-plugin-Cloud-Azure-Management-Discover-Api
```

</TabItem>
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">

```bash
apt install centreon-plugin-cloud-azure-management-discover
```

</TabItem>
<TabItem value="CentOS 7" label="CentOS 7">

```bash
yum install centreon-plugin-Cloud-Azure-Management-Discover-Api
```

</TabItem>
</Tabs>

## Troubleshooting

### The Azure credentials have changed and the Plugin does not work anymore

> The Plugin is using a cache file to keep connection information and avoid an authentication at each call. 

Reviewer: ```suggestion
The plugin is using a cache file to keep connection information and avoid authenticating at each call. 
```

If some of the authentication parameters change, you must delete the cache file. 

> The cache file can be found within  ```/var/lib/centreon/centplugins/``` folder with a name similar to azure_api_`<md5>_<md5>_<md5>_<md5>`.

Reviewer: ```suggestion
The cache file can be found in the ```/var/lib/centreon/centplugins/``` folder with a name similar to azure_api_`<md5>_<md5>_<md5>_<md5>`.
```
